portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFilters.testcase

```diff
@@ -1830,6 +1830,81 @@ definition {
 		}
 	}
 
+	@description = "LPS-188333. Confirm that changes in the filters are not autosave"
```

Comment:
This description is not correct. You are editing and deleting a filter, so you can confirm the changes are autosaved